resources/build/ci/trigger-build-bot.inc.sh

```diff
@@ -105,7 +132,7 @@ function build_bot_check_messages() {
 # Note that this function assumes that inputs are sanitized, see
 # build_bot_check_messages
 #
-function build_bot_update_commands() {
+function _build_bot_update_commands() {
```

Comment:
This was an internal function which should have been underscored from the beginning.